.babelrc

```diff
@@ -0,0 +1,16 @@
+{
+    "plugins": [
+        "@babel/plugin-syntax-dynamic-import",
```

Comment:
ohhhh... Babel 7? 👍

Reply:
Yes indeed. It's got that new transpiler smell.